.changeset/odd-ads-love.md

```diff
@@ -0,0 +1,10 @@
+---
+"emails-and-messages": patch
+"app-avatax": patch
+"cms-v2": patch
+"search": patch
+"app-taxjar": patch
+"taxes": patch
+---
+
+Use GraphQL OTEL exchange
```

Comment:
Please improve changeset messages. Changesets are not commits. We should ad 1-2 sentences what, why, what is the result after this is releases